Client/Application/AppDelegate.swift

```diff
@@ -273,6 +272,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         }
         
         AdblockResourceDownloader.shared.startLoading()
+        Preferences.General.isFirstLaunch.value = false
```

Comment:
I don't think we should move this, if we ran into some weird issue, it could end up duplicating all default favorites. I think that is the only real thing that uses the `isFirstLaunch` bool.